### ISSUE TITLE
www.gatsbyjs.org/%202020 is showing a white blank page

### ISSUE THREAD

**Issue by dengdengyee** [gatsbyjs/gatsby#19474]
## Description

www.gatsbyjs.org/%202020 is showing a white blank page.
It is not redirected to home!

### Steps to reproduce

goto www.gatsbyjs.org/%202020

### Expected result

should be redirected to www.gatsbyjs.org

### Actual result

"page not found" appears like 1 sec? and just disappears and then white blank page.

### Environment
Both chrome and safari does this.

**Comment by davidbailey00** [comment]
Hi there, thanks for opening this issue! I can see that there's an issue in that this page becomes blank; however, I'm struggling to understand why this should redirect to the homepage. Could you tell me the significance of this link in particular and why it should redirect home?

**Comment by dengdengyee** [comment]
My bad, I didn't know that it doesn't redirect to home when a corresponding page not exists. I prejudged 
 it should be redirected to home. 

By the way, I found out that opening this page https://www.gatsbyjs.org/%202020  with the network throttling configured to slow 3g, as soons as `app-3d702498a53c602c51a4.js` this one loads, the normal "page not found" wiped out and It became to the white blank page.

Hope this is helpful for you.
